tableaudocumentapi/multilookup_dict.py

```diff
@@ -1,8 +1,14 @@
+import pprint
```

Comment:
oh, yes, dang it, I usually do the import inline,  so I don't make this mistake.